catkin_ws/src/vision/src/point_cloud.py

```diff
@@ -62,8 +62,8 @@ def convert_from_uvd(color, z_map):
         xyz_rbg_img[:, :, 0] = z_map
         xyz_rbg_img[:, :, 1] = x_map
         xyz_rbg_img[:, :, 2] = y_map
-        point_cloud_img = bridge.cv2_to_imgmsg(np.float32(xyz_rbg_img[:,:,:3]), "bgr8")
-        point_cloud_img_pub.publish(point_cloud_img)
+        # point_cloud_img = bridge.cv2_to_imgmsg(np.float32(xyz_rbg_img[:,:,:3]), "bgr8")
```

Comment:
was this causing errors? I'm pretty sure this is important for the vision package, it uses this topic to get point cloud information. It should be replaced with listening directly to the point cloud topic and extracting rgb/xyz images directly but for now this should stay

Reply:
the bug can be fixed by removing the optional "bgr8" argument